dev/docker/ballista.dockerfile

```diff
@@ -44,6 +44,18 @@ RUN cargo install cargo-chef --version 0.1.34
 #RUN cargo chef cook $RELEASE_FLAG --recipe-path recipe.json
 
 FROM base as builder
+
+RUN mkdir /tmp/protoc
+WORKDIR /tmp/protoc
+
+RUN export PROTO_ZIP="protoc-21.4-linux-x86_64.zip" && \
+  curl -LO https://github.com/protocolbuffers/protobuf/releases/download/v21.4/$PROTO_ZIP && \
```

Comment:
It might be good to interpolate `21.4` into it's own variable, but better to fix these than not fix!